packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_dashboard/views/dashboard_page.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:provider/provider.dart';
+import '../../base/extensions/error_model_field_translations.dart';
+import '../../base/models/errors/error_model.dart';
+import '../blocs/dashboard_bloc.dart';
+
+class DashboardPage extends StatelessWidget {
+  const DashboardPage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+        appBar: AppBar(
+          actions: [
+            IconButton(
+              icon: const Icon(Icons.refresh),
```

Comment:
This icon can be added through the design system

Reply:
Done